benchmarks/bench.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const minimist = require('minimist')
+const bench = require('fastbench')
+const net = require('net')
+const childProcess = require('child_process')
+const path = require('path')
+const parallel = require('fastparallel')({
+  results: false
+})
+const pull = require('pull-stream')
+const toPull = require('stream-to-pull-stream')
+
+const argv = minimist(process.argv.slice(2), {
+  boolean: 'child',
+  default: {
+    child: true,
+    port: 3000,
+    host: 'localhost',
+    lib: 'pull-mplex'
+  }
+})
+
+function buildPingPong (cb) {
+  let child
+  let dialer
+  const mplex = require(argv.lib || 'pull-mplex')
+
+  if (argv.child) {
+    child = childProcess.fork(path.join(__dirname, 'mplex-echo.js'), {
+      stdio: 'inherit'
```

Comment:
I think this is what is causing clinic to crash.

Reply:
That makes sense, I can just consolidate the test to a single file, it doesn't really need to be separated.